Gemfile.lock

```diff
@@ -302,3 +319,6 @@ DEPENDENCIES
   turbolinks
   uglifier (>= 1.3.0)
   xray-rails
+
+BUNDLED WITH
```

Comment:
Ahh, no matter how many times I've done this I still forget, what a f-g nuisance.